src/application.js

```diff
@@ -5,9 +5,33 @@
 Marionette.Application = Marionette.Object.extend({
   cidPrefix: 'mna',
 
+  rootEl: 'body',
+
+  constructor: function(options) {
+    options = options || {};
+
+    this._initRootRegion(options);
+
+    Marionette.Object.prototype.constructor.apply(this, arguments);
+  },
+
+  _initRootRegion: function(options) {
+    var rootEl = options.rootEl || this.rootEl;
+
+    rootEl = Marionette._getValue(rootEl, this, options);
+
+    // rootEl is optional
+    if (rootEl) {
+
+      var RootRegion = Marionette.Region.extend({
+        el: rootEl
+      });
+
+      this.rootEl = new RootRegion();
```

Comment:
one other thought.. there's not any region events we'll want to proxy here?  it kinda doesn't seem like it.  But we could add a `before:show:rootView` and `show:rootView`  any value?